src/core/cff_parser.js

```diff
@@ -1413,7 +1394,28 @@ var CFFCompiler = (function CFFCompilerClosure() {
     compileNameIndex: function CFFCompiler_compileNameIndex(names) {
       var nameIndex = new CFFIndex();
       for (var i = 0, ii = names.length; i < ii; ++i) {
-        nameIndex.add(stringToBytes(names[i]));
+        var name = names[i];
+        var sanitizedName = [];
```

Comment:
IIIRC, specifying array length doesn't really matter. I also looked back at a few of Nicholas' patches and he seemed to also use plain [] and push as well.

Reply:
In this case the lengths ought to be way too short for it to matter, but in the general case it might help, see e.g. PR #5191.
Anyway, I don't really care much either way in this case :-)